### PR TITLE
Use standard payment process

### PR DESCRIPTION
**Pull request by imollov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,23 +112,19 @@ npx hardhat run scripts/deploy_staging.ts --network <network>
...
 ```bash
 npx hardhat run scripts/deploy_topup.ts --network <network>
 ```
 
-- The deployed contract must be initialized with `vestId` and `upkeepId` by calling the respective setter functions
-  - To acquire `vestId`, a new application must be [submitted to MakerDAO](https://forum.makerdao.com/t/mip63-maker-keeper-network/12091)
-  - The `upkeepId` is available after registering a new Upkeep
-
-3. Finally, to enable auto refunding of the `DssCronKeeper` upkeep, call `setUpkeepRefunder(address)` and pass the address of the deployed `DssVestTopUp`
+The deployed contract must be initialized with `NetworkPaymentAdapter` address by calling `setPaymentAdapter` from the contract owner account.
```
Comment (ivaylonikolov7) [inline README.md:123]: Why don't we add this in the deploy_topup script?

File: test/DssVestTopUp.test.ts [hunk 2]
```diff
@@ -86,199 +65,114 @@ describe("DssVestTopUp", function () {
...
+      const expectedDaiAmountIn = topUpAmount;
+      const expectedLinkAmountOut = expectedDaiAmountIn;
 
-    it("should update pool fee", async function () {
-      const oldPoolFee = await topUp.uniswapPoolFee();
-      await topUp.setUniswapPoolFee(10000);
-      const newPoolFee = await topUp.uniswapPoolFee();
-      expect(oldPoolFee).to.not.eq(newPoolFee);
+      await expect(topUp.refundUpkeep())
+        .to.emit(topUp, "SwappedDaiForLink")
```
Comment (ivaylonikolov7) [inline test/DssVestTopUp.test.ts:128]: Isnt it more reliable to check `balanceOf` of DaiToken ?

**Comment by ivaylonikolov7** [comment]
Also what about the deploy_staging script? Are we not gonna update that?

File: test/DssVestTopUp.test.ts (hunk 2 above)
Comment (imollov) [reply test/DssVestTopUp.test.ts]: It is but not sure if it's worth doing it for a mock contract.

File: .prettierrc [hunk 3]
```diff
@@ -1 +1,15 @@
...
+        "printWidth": 120,
+        "tabWidth": 4,
+        "useTabs": false,
+        "singleQuote": false,
+        "bracketSpacing": false,
+        "explicitTypes": "always"
+      }
+    }
+  ]
+}
```
[the remark this answers is not in the document]
Comment (imollov) [reply .prettierrc:15]: I added it in the description of the commit because it will be very hard to separate it right now.

**Comment by imollov** [comment]
> Also what about the deploy_staging script? Are we not gonna update that?

The script was deleted so this comment is outdated.

**Review by ivaylonikolov7 — APPROVED** [review]

**Pull request merged**